apps/web/composables/usePayPal/usePayPal.ts

```diff
@@ -0,0 +1,103 @@
+import { loadScript as loadPayPalScript } from '@paypal/paypal-js';
+import { PayPalExecuteParams } from '@plentymarkets/plentymarkets-sdk/packages/api-client/server';
+import { paypalGetters } from '~/getters/paypalGetters';
+import { useSdk } from '~/sdk';
+import type {
+  UsePayPalMethodsReturn,
+  createTransaction,
+  executeOrder,
+  LoadScript,
+  UsePayPalState,
+  approveOrder,
+} from './types';
+
+/**
+ * @description Composable for paypal.
+ * @returns {@link UsePayPalMethodsReturn}
+ * @example
+ * const { loadScript } = usePayPal();
+ */
+export const usePayPal: UsePayPalMethodsReturn = () => {
+  const state = useState<UsePayPalState>('usePayPal', () => ({
+    PayPalScript: null,
+    order: null,
+  }));
+
+  /**
+   * @description Function for fetching the category tree.
+   * @example
+   * loadScript(currency: string);
+   */
+  const loadScript: LoadScript = async (currency: string) => {
+    if (paypalGetters.getClientId()) {
+      try {
+        state.value.PayPalScript = await loadPayPalScript({
+          clientId: paypalGetters.getClientId() ?? '',
+          currency: currency,
+          dataPartnerAttributionId: 'plentysystemsAG_Cart_PPCP',
+        });
+        return state.value.PayPalScript;
+      } catch {
+        // console.error('failed to load the PayPal JS SDK script', error);
```

Comment:
**question**: Can we replace the commented out console error with a TODO of how we ultimately intend to handle the error?